e2e/completeTransferFlow.test.ts

```diff
@@ -0,0 +1,157 @@
+import { UserFactory } from "../libs/user";
+import { Balance, User } from "../libs/user/types";
+
+let user1: User;
+let user2: User;
+let user3: User;
+let user4: User;
+let user4Balance: any;
+let user3Balance: any;
+
+const TX_PER_BLOCK = 2;
+const DECIMALS = 1000000000;
+
+// Values for deposit
+const DEPOSIT_VALUE = 100000;
+const TRANSACTION_DEPOSIT_VALUE = DEPOSIT_VALUE / DECIMALS; // 0,0001
+const TRANSACTION_DEPOSIT_BALANCE = DEPOSIT_VALUE * DECIMALS;
+
+// Values for transfer
+const TRANSFER_VALUE = 20000;
+const TRANSACTION_TRANSFER_VALUE = TRANSFER_VALUE / DECIMALS; // 0,0001
+const TRANSACTION_TRANSFER_BALANCE = TRANSFER_VALUE * DECIMALS;
+
+const waitForTimeout = async (timeoutInMs: number) => {
+  await new Promise((resolve) => setTimeout(resolve, timeoutInMs));
+};
+
+describe("nightfall smoke tests", function () {
```

Comment:
Isn't this the same [smoke test](https://github.com/maticnetwork/nightfall-cicd/pull/4/files#diff-b9cadb85c6ef3adabb3adf08646b393e032c4c3575cea5766c2f361348c3daab) in ci/cd repo?